### ISSUE TITLE
Fix ParseCIDR - error while deploying to remote machines with multiple IPs (like GCP)

### ISSUE THREAD

**Issue by mydatascience** [lf-edge/eden#231]
```
./eden eve ssh
FATA[0000] invalid CIDR address: 10.138.0.51
```
This happens because the IP line is not parsed correctly 

```
for _, nw := range lastDInfo.GetDinfo().Network {
						for _, addr := range nw.IPAddrs {
							ip, _, err := net.ParseCIDR(addr)
							if err != nil {
								log.Fatal(err)
							}
```

**Comment by mydatascience** [comment]
Fixed at #228

**Comment by mydatascience** [comment]
No more an issue for me
